Assets/MixedRealityToolkit/Utilities/BuildAndDeploy/UwpAppxBuildTools.cs

```diff
@@ -170,41 +170,56 @@ public static async Task<bool> BuildAppxAsync(string productName, bool forceRebu
 #endif
 
             // Ensure that the generated .appx version increments by modifying Package.appxmanifest
-            if (!SetPackageVersion(incrementVersion))
+            if (!SetPackageVersion(buildDirectory, incrementVersion))
             {
                 Debug.LogError("Failed to increment package version!");
                 return IsBuilding = false;
             }
 
+            int msBuildExitCode;
             // Now do the actual appx build
-            var processResult = await new Process().StartProcessAsync(
-                msBuildPath,
-                $"\"{solutionProjectPath}\" /t:{(forceRebuildAppx ? "Rebuild" : "Build")} /p:Configuration={buildConfig} /p:Platform={buildPlatform} /verbosity:m",
-                true);
-
-            if (processResult.ExitCode == 0)
-            {
-                Debug.Log("Appx Build Successful!");
-            }
-            else if (processResult.ExitCode == -1073741510)
-            {
-                Debug.LogWarning("The build was terminated either by user's keyboard input CTRL+C or CTRL+Break or closing command prompt window.");
-            }
-            else if (processResult.ExitCode != 0)
+            using (var process = new Process())
             {
-                Debug.LogError($"{PlayerSettings.productName} appx build Failed! (ErrorCode: {processResult.ExitCode})");
+                process.StartInfo = new ProcessStartInfo
+                {
+                    FileName = msBuildPath,
+                    CreateNoWindow = true,
+                    UseShellExecute = false,
+                    RedirectStandardOutput = false,
+                    RedirectStandardError = false,
+                    Arguments = $"\"{solutionProjectPath}\" /t:{(forceRebuildAppx ? "Rebuild" : "Build")} /p:Configuration={buildConfig} /p:Platform={buildPlatform} /verbosity:m -fl -flp:logfile=appxbuild.log;verbosity=diagnostic",
```

Comment:
```suggestion
                    Arguments = $"\"{solutionProjectPath}\" /t:{(forceRebuildAppx ? "Rebuild" : "Build")} /p:Configuration={buildConfig} /p:Platform={buildPlatform} /verbosity:m",
```
Left over from testing